packages/commonwealth/client/styles/pages/manage_community/chain_metadata_rows.scss

```diff
@@ -4,7 +4,6 @@
   display: flex;
   flex-direction: column;
```

Comment:
Maybe `justify-content: space-between;` here instead of `flex-direction: column`? Current layout looks kinda funky

Reply:
I think where it's at is the best compromise for right now. But agree that it's not optimal.